content/documentation/getting-started/index.adoc

```diff
@@ -299,6 +299,57 @@ The login option can be specified in the Kiali CR when installing Kiali. For ins
 auth:
   strategy: openshift
 ----
+*ldap*: This option allows you to set the authentication strategy so that the user is validated using the LDAP server, then an authentication token is generated. A sample configuration for LDAP is given below
+----
+auth:
+  strategy: ldap
+  ldap:
+    ldap_host: example.com
+    ldap_port: 123
+    ldap_use_ssl: false
+    ldap_insecure_skip_verify: true
+    ldap_user_filter: (cn=%s)
+    ldap_group_filter: (cn=%s)
+    ldap_attributes: ['memberOf', 'cn', 'mail']
```

Comment:
This has been removed in the kiali PR - so I assume these docs need to reflect the new code:

https://github.com/kiali/kiali/pull/1253/files/9e7e649e96b3201d08b2f293b9a1444adb0ab6ad..5320578573c49739e99a2541df8beb50254f8d07#diff-3baf47c64847a8fb8aaa8cc2e088513bL247